src/commands/functions/functions-create.js

```diff
@@ -565,6 +628,41 @@ const installAddons = async function (command, functionAddons, fnPath) {
   return Promise.all(arr)
 }
 
+const registerEFInToml = async (funcName) => {
+  if (!fs.existsSync('netlify.toml')) {
+    log(`${NETLIFYDEVLOG} \`netlify.toml\` file does not exist yet. Creating it...`)
+  }
+
+  let { funcPath } = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'funcPath',
+      message: `What route do you want your edge function to be invoked on? \nRoute can be configured by editing your \`netlify.toml\` file.`,
+      default: '/test',
+      validate: (val) => Boolean(val),
+      // Make sure route isn't undefined and is valid
+      // Todo: add more validation?
+    },
+  ])
+
+  // Make sure path begins with a '/'
+  if (funcPath[0] !== '/') {
+    funcPath = `/${funcPath}`
+  }
+
+  const functionRegister = `\n\n[[edge_functions]]\nfunction = "${funcName}"\npath = "${funcPath}"`
+
+  try {
+    fs.promises.appendFile('netlify.toml', functionRegister)
+    log(
+      `${NETLIFYDEVLOG} Function '${funcName}' registered for route \`${funcPath}\`. To change, edit your \`netlify.toml\` file.`,
+    )
+  } catch {
+    log(`${NETLIFYDEVERR} Unable to register function. Please check your \`netlify.toml\` file.`)
```

Comment:
I think we should use the `error` helper here instead of `log`. It also means you don't need to explicitly call `process.exit`.